src/main/resources/s3capabilities.json

```
@@ -1,284 +1,71 @@
-    "put_bucket_metrics_configuration": {},
-    "delete_bucket_metrics_configuration": {},
-    "get_bucket_replication": {},
-    "put_bucket_replication": {},
-    "delete_bucket_replication": {},
-    "get_bucket_accelerate": {},
-    "put_bucket_accelerate": {},
-    "list_bucket_inventory_configuration": {},
-    "get_bucket_inventory_configuration": {},
-    "put_bucket_inventory_configuration": {},
-    "delete_bucket_inventory_configuration": {},
-    "list_bucket_analytics_configuration": {},
-    "get_bucket_analytics_configuration": {},
-    "put_bucket_analytics_configuration": {},
-    "delete_bucket_analytics_configuration": {},
-    "get_public_access_block": {},
-    "put_public_access_block": {},
-    "delete_public_access_block": {},
-    "delete_objects": {},
-    "upload_part_copy": {},
-    "get_object_torrent": {},
-    "restore_object": {},
-    "select_object_content": {}
+     "delete_bucket": {},
+     "delete_bucket_cors": {},
+     "delete_bucket_encryption": {},
+     "delete_bucket_lifecycle": {},
+     "delete_bucket_metrics_configuration": {},
+     "delete_bucket_policy": {},
+     "delete_bucket_replication": {},
+     "delete_bucket_website": {},
+     "get_bucket_list_objects_version1": {},
+     "get_bucket_list_objects_version2": {},
+     "get_bucket_acl": {},
+     "get_bucket_cors": {},
+     "get_bucket_encryption": {},
+     "get_bucket_lifecycle": {},
+     "get_bucket_location": {},
+     "get_bucket_metrics_configuration": {},
+     "get_bucket_notification": {},
+     "get_object_lock_configuration": {},
+     "get_bucket_object_versions": {},
+     "get_bucket_policy": {},
+     "get_bucket_replication": {},
+     "get_bucket_versioning": {},
+     "get_bucket_website": {},
+     "head_bucket": {},
+     "list_bucket_metrics_configurations": {},
+     "list_multipart_uploads": {},
+     "put_bucket": {},
+     "put_bucket_acl": {},
+     "put_bucket_analytics_configuration": {},
+     "put_bucket_cors": {},
+     "put_bucket_encryption": {},
+     "put_bucket_lifecycle": {},
+     "put_bucket_metrics_configuration": {},
+     "put_bucket_notification": {},
+     "put_object_lock_configuration": {},
+     "put_bucket_policy": {},
+     "put_bucket_replication": {},
+     "put_bucket_versioning": {},
+     "put_bucket_website": {},
+     "delete_objects": {},
+     "delete_object": {},
+     "delete_object_tagging": {},
+     "get_object": {},
+     "get_object_acl": {},
+     "get_object_legal_hold": {},
+     "get_object_retention": {},
+     "get_object_tagging": {},
+     "head_object": {},
+     "put_object": {},
+     "put_object_copy": {},
+     "put_object_acl": {},
+     "put_object_legal_hold": {},
+     "put_object_retention": {},
+     "put_object_tagging": {},
+     "abort_multipart_upload": {},
+     "complete_multipart_upload": {},
+     "initiate_multipart_upload": {},
+     "list_parts": {},
+     "upload_part": {},
+     "upload_part_copy": {}
   }
-}
+  }
```

```suggestion
}
```

A space is not needed for the last `}`